test/libyul/yulControlFlowGraph/function.yul

```diff
@@ -0,0 +1,54 @@
+{
+    function f(a, b) -> r {
+        let x := add(a,b)
+        r := sub(x,a)
+    }
+    function g() {
+        sstore(0x01, 0x0101)
+    }
+    function h(x) {
+        h(f(x, 0))
+        g()
+    }
+    function i() -> v, w {
+        v := 0x0202
+        w := 0x0303
+    }
+    let x, y := i()
+    h(x)
+    h(y)
+}
+// ----
+// Block 0:
```

Comment:
Are the `RET[f]` the function calls?

Reply:
Probably should have documented these somewhere :-). ``RET[f]`` is the return label of a specific call to ``f`` (it's a bit unfortunate that it's not easy to refer to some specific call more precisely than just using the function name).
For the record (it won't occur in this PR, but in later ones): There is also ``RET`` (without index), which refers to the return label of a function *within* the body of the function (i.e. not associated with a specific call and without a statically known value).